cfpb/design-manual
# Updates the project to use the latest release from Capital Framework.

Updates the project to use the latest release from Capital Framework.

## Changes

- Updates cf-forms to 5.2.0

## Testing

- Run `npm start`
- Open http://localhost:4000/page-components/expandables.html and make sure the expandables match the spec correctly

## Screenshots

<img width="806" alt="screen shot 2018-07-19 at 11 55 09 am" src="https://user-images.githubusercontent.com/1280430/42958414-12d15c18-8b4b-11e8-8a6e-db89ba756274.png">

<img width="805" alt="screen shot 2018-07-19 at 11 55 21 am" src="https://user-images.githubusercontent.com/1280430/42958416-141da86a-8b4b-11e8-931d-5f14d9efd710.png">
